packages/template-retail-react-app/package.json

```diff
@@ -83,7 +83,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "270 kB"
+            "maxSize": "280 kB"
```

Comment:
How big of an increase in bundle size are we looking at? have you investigated maybe doing something a little more lightweight?

Take a look at the following resources that might be simpler alternatives.

https://geshan.com.np/blog/2022/01/nodejs-uuid/#node.js-uuid-with-crypto-module
https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid

Reply:
@bendvc I checked the uuid from crypto module before, it only works on server-side, not client.